packages/app-defaults/src/defaults/apis.ts

```diff
@@ -64,10 +64,11 @@ export const apis = [
   createApiFactory({
     api: discoveryApiRef,
     deps: { configApi: configApiRef },
-    factory: ({ configApi }) =>
-      UrlPatternDiscovery.compile(
+    factory: ({ configApi }) => {
+      return UrlPatternDiscovery.compile(
         `${configApi.getString('backend.baseUrl')}/api/{{ pluginId }}`,
-      ),
+      );
+    },
```

Comment:
Can revert this completely right? Just to avoid changes

Reply:
Yup, will update.